schemas/Manual.locations.schema.json

```diff
@@ -46,56 +42,59 @@
                     },
                     "uniqueItems": true
                 },
-                "place_item_category": {
-                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
+                "region": {
+                    "description": "(Optional) The name of the Region this location is part of.",
+                    "type": "string"
+                },
+                "place_item": {
+                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
```

Comment:
Should be "list of items" not "list of item".

Also "will choose 1 at random" sounds better than "will only randomly choose 1"

Reply:
Tweaked this as well.